lib/ansible/modules/cloud/azure/azure_rm_sql_server.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_sql_server
+version_added: "2.5"
+short_description: Manage an Server.
+description:
+    - Create, update and delete an instance of Server.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    name:
+        description:
+            - The name of the server.
+        required: True
+    location:
+        description:
+            - Resource location.
+        required: True
+    tags:
+        description:
+            - Resource tags.
+        required: False
+    identity:
+        description:
+            - The Azure Active Directory identity of the server.
+        required: False
+        suboptions:
+            type:
+                description:
+                    - "The identity type. Set this to 'SystemAssigned' in order to automatically create and assign an Azure Active Directory principal for th
+                       e resource. Possible values include: 'SystemAssigned'"
+                required: False
+    administrator_login:
+        description:
+            - Administrator username for the server. Once created it cannot be changed.
+        required: False
+    administrator_login_password:
+        description:
+            - The administrator login password (required for server creation).
+        required: False
+    version:
+        description:
+            - The version of the server.
+        required: False
+
+extends_documentation_fragment:
+    - azure
+    - azure_tags
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+      - name: Create (or update) Servers
+        azure_rm_sql_server:
+          resource_group: "{{ resource_group }}"
+          name: zims-server
+          location: westus
+          tags: "{{ tags }}"
+          identity:
+            type: "{{ type }}"
+          administrator_login: mylogin
+          administrator_login_password: Testpasswordxyz12!
+          version: "{{ version }}"
```

Comment:
Would be nice if you can actually show an example of a valid version value, nothing worse than seeing that it can be specified but not knowing the form it takes.